Makefile

```diff
@@ -33,100 +33,58 @@ ifneq ($(TAGS),)
 BUILDTAGS = -tags '$(TAGS)'
 endif
 
+
+.PHONY: build
 build: nt-connect
 
+.PHONY: dist
+dist: DESTDIR=dist/nt-connect_$(VERSION)_$(GOOS)_$(GOARCH)
+dist:
+	@make DESTDIR=$(DESTDIR) -B install
+	tar -C $(DESTDIR) -czf $(DESTDIR).tar.gz .
```

Comment:
FYI all files are uploaded to the pipeline Artifacts:

![image](https://github.com/NorthernTechHQ/nt-connect/assets/3168644/5acb1703-35b0-4c31-8060-09f4402b2911)

Consider removing  `$(DESTDIR) `

Reply:
Ahh, right! I'll instead install into a temporary directory and only create the tarballs in `dist`.

Reply:
I just made it really simple by adding `--remove-files` to the tar command so it removes them as they are added to the archive.